### PR TITLE
[Not For Main]Audio: Volume: Add bypass functions

### PR DESCRIPTION
**Pull request by andrula-song**
Add bypass functions for volume. If gain of all channels equals 0dB, then we use bypass functions to transport the stream data.

### CODE REVIEW

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by aiChaoSONG** [comment]
SOFCI TEST

File: src/audio/module_adapter/module/volume/volume.c [hunk 1]
```diff
@@ -267,15 +267,18 @@ static inline int32_t volume_windows_fade_ramp(struct vol_data *cd, int32_t ramp
  */
 
 /* Note: Using inline saves 0.4 MCPS */
-static inline void volume_ramp(struct vol_data *cd)
+static inline void volume_ramp(struct processing_module *mod)
```
Comment (singalsu) [inline src/audio/module_adapter/module/volume/volume.c:270]: The checkpatch fail is from commit message typo "If tha gain".

Comment (andrula-song) [reply src/audio/module_adapter/module/volume/volume.c:270]: thanks, I am waiting the CI test result. And then push again.

**Pull request merged**